conans/client/tools/system_pm.py

```diff
@@ -2,10 +2,15 @@
 from conans.client.runner import ConanRunner
 from conans.client.tools.oss import OSInfo
 from conans.errors import ConanException
+from conans.util.env_reader import get_env
 
 _global_output = None
 
 
+class SystemPackageToolInvalidMode(ConanException):
```

Comment:
Why a new exception class? it is not used anywhere

Reply:
Not it's only to report this problem, do you want to use ConanException instead?

Reply:
Yes